### DISCUSSION TITLE
[General] <Title>Call exception error trying to run paymaster-transaction.ts

### DISCUSSION THREAD

**Discussion by umutsatir** [zkSync-Community-Hub/zksync-developers#569] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/34773592/abd6c597-5c2d-4095-9881-fdcec145354e)
I got this error trying to use paymaster script.

**Comment by uF4No** [comment, marked as answer]
Hi @umutsatir 

As mentioned in the tutorial, we've identified an issue with Remix and this script. Please use Atlas instead for this part of the tutorial. 

We'll update it in the docs once this issue is resolved.
